src/server/game/Handlers/CharacterHandler.cpp

```diff
@@ -1405,6 +1406,13 @@ void WorldSession::HandleCharCustomize(WorldPacket& recvData)
         return;
     }
 
+    //Prevent To Rename Character (Poriya)
```

Comment:
it's my name :D i use it just for fast find my coding line . i should remove this too ...